runtime/src/accounts_index.rs

```diff
@@ -212,9 +216,16 @@ impl AccountMapEntryMeta {
 }
 
 #[derive(Debug, Default)]
+/// one entry in the in-mem accounts index
+/// key is Pubkey
+/// this is the value
 pub struct AccountMapEntryInner<T> {
+    /// number of active slots that contain >= 1 instances of account data for this pubkey
```

Comment:
active -> alive, where alive represents a slot that has not yet been removed by clean via `AccountsDB::clean_stored_dead_slots()` for containing no up to date account information

Reply:
updated